examples/plot_simulate_evoked_newapi.py

```diff
@@ -0,0 +1,150 @@
+# inside it. The default behaviour of Network is to add and instantiate six
+# 'default' drives, but we will suppress that by setting the
+# `initialise_hnn_drives`-argument to `False`.
+net = Network(params, initialise_hnn_drives=False)
+net.plot_cells()
+
+###############################################################################
+# The network of cells is now defined, to which we add external drives as
+# required. Weights are prescribed separately for AMPA and NMDA receptors
+# (receptors that are not used can be omitted or set to zero)
+
+# Distal evoked drive
+weights_ampa = {'L2_basket': 0.006562, 'L2_pyramidal': .000007,
+                'L5_pyramidal': 0.142300}
+weights_nmda = {'L2_basket': 0.019482, 'L2_pyramidal': 0.004317,
+                'L5_pyramidal': 0.080074}
+dispersion_time = {'L2_basket': 0.1, 'L2_pyramidal': 0.1,
+                   'L5_pyramidal': 0.1}
+net.add_evoked_drive(
+    'evdist1', mu=63.53, sigma=3.85, numspikes=1, weights_ampa=weights_ampa,
+    weights_nmda=weights_nmda, location='distal', seedcore=2,
+    space_constant=3., dispersion_time=dispersion_time)
+
+# First proximal evoked drive
+weights_ampa = {'L2_basket': 0.08831, 'L2_pyramidal': 0.01525,
+                'L5_basket': 0.19934, 'L5_pyramidal': 0.00865}
+weights_nmda = {'L2_basket': 0., 'L2_pyramidal': 0.,
+                'L5_basket': 0., 'L5_pyramidal': 0.}  # all weights zero
+dispersion_time = {'L2_basket': 0.1, 'L2_pyramidal': 0.1,
+                   'L5_basket': 1., 'L5_pyramidal': 1.}
+net.add_evoked_drive(
+    'evprox1', mu=26.61, sigma=2.47, numspikes=1, weights_ampa=weights_ampa,
+    weights_nmda=weights_nmda, location='proximal', seedcore=2,
+    space_constant=3., dispersion_time=dispersion_time)
+
+# Second proximal evoked drive
+weights_ampa = {'L2_basket': 0.000003, 'L2_pyramidal': 1.438840,
+                'L5_basket': 0.008958, 'L5_pyramidal': 0.684013}
+weights_nmda = {}  # all weights zero: pass an empty dict to the add-method
+dispersion_time = {'L2_basket': 0.1, 'L2_pyramidal': 0.1,
+                   'L5_basket': 1., 'L5_pyramidal': 1.}
+net.add_evoked_drive(
+    'evprox2', mu=137.12, sigma=8.33, numspikes=1, weights_ampa=weights_ampa,
+    weights_nmda=weights_nmda, location='proximal', seedcore=2,
+    space_constant=3., dispersion_time=dispersion_time)
+
+###############################################################################
+# Now let's simulate the dipole, running 2 trials with the Joblib backend.
+# To run them in parallel we could set n_jobs to equal the number of trials.
+from hnn_core import JoblibBackend
+
+with JoblibBackend(n_jobs=1):
+    dpls = simulate_dipole(net, n_trials=2, postproc=True)
+
+###############################################################################
+# and then plot it
+import matplotlib.pyplot as plt
+fig, axes = plt.subplots(2, 1, sharex=True, figsize=(6, 6))
+plot_dipole(dpls, ax=axes[0], layer='agg', show=False)
```

Comment:
sorry to be party pooper but the output looks different than the other example :( I think we should just replace the old example if it's good by @rythorpe and @ntolley

Reply:
Party's just getting started ;) I felt that way at first too, but then realised it's precisely to be expected: the `_new` version only has the drives that are actually in use. The old example has `common`, `extgauss` and `extpoiss` feeds even though they aren't used. The fact that GIDs 270 and 271 get assigned to `common` means that the event time distributions of the subsequent `ev*`-drives are affected. Hence the _qualitative_ difference between old and new dipole traces.

Tests pass and CI-artefacts are otherwise OK because most tests still use the old API.

We're closing in on the time when examples get detached from using HNN GUI as gold standard...

Reply:
okay I see. Just to be doubly sure, would you mind adding those feeds with 0 connectivity and see if you can reproduce? We can remove it before merging. And I would add an entry in the "BUG" section of `whats_new.rst` as it seems the old API was technically using a different seed than was intended. I agree we're getting closer to be the gold standard but even more the reason to be extra careful, specially in a large PR :)

Reply:
Actually, I'm not sure I understand. Since each feed has a separate seed, shouldn't they be independent of each other in terms of event_times?

Reply:
Added "dud"-drives to latest push. Results are identical to `master`/GUI. That's because the first seed in 'evdist1' is `2 + 272`, where 272 is the GID of the first `_ArtificialCell` in that drive. Without the two empty `bursty` drives, the starting gid would be 270 and the seed `2 + 270`.

Reply:
Fantastic, thanks for checking. You can replace the old example now. When rebasing, if you can keep this commit where we checked the result exactly matches, that would be great.